node/node_test.go

```diff
@@ -702,10 +708,11 @@ func TestNodeSetEventSink(t *testing.T) {
 	t.Cleanup(cleanup(ns))
 }
 
-func state(t *testing.T, nVals int, height int64) (sm.State, dbm.DB, []types.PrivValidator) {
+func state(t *testing.T, nVals int, height int64) (sm.State, dbm.DB, []types.PrivValidator, []crypto.Address) {
```

Comment:
You should be able to get this from privval, is there a reason to return this separately?

Reply:
Good question. I think this is an artifact from an earlier iteration where, for some reason, I wasn't getting the same addresses from the privvals as what I expected. I've removed this now as it's no longer necessary given how the test's structured.